src/components/gui/gui.jsx

```diff
@@ -240,7 +240,7 @@ const GUIComponent = props => {
                             ) : null}
                         </Box>
 
-                        <Box className={styles.stageAndTargetWrapper}>
+                        <Box className={classNames(styles.stageAndTargetWrapper, styles[stageSize])}>
```

Comment:
I'm not familiar with this syntax... this looks to me like it would pick either:
- `styles.stageAndTargetWrapper` and `'.large'`, or
- `styles.stageAndTargetWrapper.large` but not `styles.stageAndTargetWrapper`

My guess is that my interpretation is incorrect, but could you explain what's actually going on here?